Makefile

```diff
@@ -1,41 +1,48 @@
 YARPC=yarpc
 TEST_DIR=tests
 PYTHON?=python3
-PIP?=pip
+PIP3?=pip3
```

Comment:
Won't it cause error on windows?

Reply:
I don't think this will fail in Windows?